Exercise-14/index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Regex exercise</title>
+
+    <!-- external fonts -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500&display=swap" rel="stylesheet" />
+
+    <!-- external css -->
+    <link rel="stylesheet" href="assets/css/styles.css" />
+  </head>
+  <body>
+    <!-- header section start -->
+    <header class="logo-container">
+      <img src="assets/Sirius_CDW_Cobranded_PMS2747.svg" alt="sirius-logo" />
+    </header>
+    <!-- header section end -->
+
+    <!-- main section start -->
+    <main>
+      <form action="post" id="paymentForm" class="payment-form">
+        <p class="form-title">BILLING DETAILS</p>
+        <div class="person-name double-input">
+          <div class="form-group">
+            <p>First Name</p>
```

Comment:
could you let me know the appropriate semantic for texts before every input element in this form

Reply:
I have seen that label is the best tag for text above input and changed it